### DISCUSSION TITLE
no ECHO, BOLD PROMPT etc in ./install script

### DISCUSSION THREAD

**Discussion by rmathar** [iraf-community/iraf#147] [Q&A] [answered]
In the installation description there is an instruction to call ./install (with options...) in the local directory after untarring the iraf.tar.gz. However, there are no commands ECHO, BOLD, PROMPT etc used but not defined in that bin/sh, and just a ton of error messages on that fact is issued. There are some lower-case equivalent standard shell command in Linuxes, but not those misspelled once. I propose to define ECHO, BOLD and so on within the install shell script so the installation can proceed.

**Comment by olebole** [comment]
The `ECHO`, `BOLD_ON` etc. functions are defined in `unix/hlib/util.sh` and sourced in the `install` script: https://github.com/iraf-community/iraf/blob/ba22d135a9f2a621bd2b5c47d5485f9a8370d2a8/install#L37-L39

It is tested to work on both Linux and macOS, like here: 
![grafik](https://user-images.githubusercontent.com/397223/89680503-5e5c9600-d8f3-11ea-8715-4bfdddc85f24.png)

Could you specify you environment (Linux? macOS? Which version and distribution?), and which IRAF version you used? What is your `/bin/sh`?

**Comment by olebole** [comment]
Is this still a problem?

**Comment by rmathar** [comment, marked as answer]
No. I forgot to use a `unset iraf` before calling `./install -s`.
A minor point is that the `mkdir` for the Local unix commands directory should have a switch `-p` to create non-existing directories.
